src/System.Linq.Parallel/tests/ParallelEnumerableTests.cs

```diff
@@ -259,6 +259,7 @@ public static IEnumerable<object[]> EmptyData()
 
         [Theory]
         [MemberData(nameof(EmptyData))]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.UapAot, "This causes assertion failure on UAPAoT")]
```

Comment:
I have opened an issue in TFS for this because its an issue for the ILC guys. I did not do any investigations into why it asserts.